### PR TITLE
trivial/minimal fix for PHP 8.2

### PR DESCRIPTION
**Pull request by remicollet**
Alternative solution is to declare each used property (connection, username, password, _filaureCallback...)

### CODE REVIEW

File: src/memcache.c [hunk 1]
```diff
@@ -733,9 +733,15 @@ PHP_MINIT_FUNCTION(memcache)
 
 	INIT_CLASS_ENTRY(ce, "MemcachePool", php_memcache_pool_class_functions);
 	memcache_pool_ce = zend_register_internal_class(&ce);
+#if PHP_VERSION_ID >= 80200
+	memcache_pool_ce->ce_flags |= ZEND_ACC_ALLOW_DYNAMIC_PROPERTIES;
```
Comment (TysonAndre) [inline src/memcache.c:737]: This looks like it'll work.

If the maintainers wanted to mark this as allowing dynamic properties, the attribute could optionally also be added to indicate that in reflection (include Zend/zend_attributes.h conditionally as well), though it isn't required

`zend_add_class_attribute(ast_node_ce, zend_ce_allow_dynamic_properties->name, 0);`

-----

If the maintainers went with declaring properties, then zend_declare_property_ex would be used
(e.g. if they wanted to warn users about typos, or didn't expect users to add any other dynamic properties in practice)

```c
	zend_declare_property_ex(memcache_pool_ce, name, value, ZEND_ACC_PUBLIC, NULL)
```

**Comment by remicollet** [comment]
Please, also add the **hacktoberfest-accepted** tag

**Comment by pmurzakov** [comment]
@remicollet it looks like a duplicate of what I did a month ago :)
https://github.com/websupport-sk/pecl-memcache/pull/102

The difference is that I fixed it in another way (by explicitly declaring properties rather than allowing them to be added dynamically)

**Comment by thesource93** [comment]
we will go for the trivial approach for now.

**Comment by itelmenko** [comment]
Could you add this release here https://pecl.php.net/package/memcache ?
Actual version is not available in pecl now

**Comment by itelmenko** [comment]
@thesource93 Please add updates to pecl

**Comment by xepozz** [comment]
Can someone tag the release?

**Comment by tomassrnka** [comment]
Sorry everyone for the delay, released with builds passing - this version has been in production for 5+ months on production env.

**Pull request merged**